### ISSUE TITLE
model_TII.py中的Unresolved reference 'BatchNorm2d'

### ISSUE THREAD

**Issue by sbjshxbxijs** [Linfeng-Tang/SeAFusion#29]
唐博士您好，我在调试时发现：model_TII.py中的BatchNorm2d（#from modules.bn import InPlaceABNSync as BatchNorm2d）被您注释掉了，BatchNorm2d没有上传到这个repo中
Unresolved reference 'BatchNorm2d'  293
Unresolved reference 'BatchNorm2d'  343

您可以提供这部分代码吗，谢谢！

**Comment by Linfeng-Tang** [comment]
应该都是使用的nn.BatchNorm2d吧，不知道你目前是否解决了这个问题。
